src/deezer/resources/user.py

```diff
@@ -48,158 +48,158 @@ class User(Resource):
     _parse_birthday = staticmethod(parse_date)
     _parse_inscription_date = staticmethod(parse_date)
 
-    def get_albums(self, **kwargs) -> PaginatedList[Album]:
+    async def get_albums(self, **kwargs) -> PaginatedList[Album]:
         """
         Get user's favorite albums.
 
         :returns: a :class:`PaginatedList <deezer.PaginatedList>`
                   of :class:`Album <deezer.Album>` instances
         """
-        return self.get_paginated_list("albums", **kwargs)
+        return await self.get_paginated_list("albums", **kwargs)
 
-    def add_album(self, album: Album | int):
+    async def add_album(self, album: Album | int):
         """
         Add an album to user's favorite albums.
 
         :param album: an :class:`~deezer.Album` instance or its ID
         :returns: a boolean that tells if the operation was successful
         """
-        return self.post_relation("albums", album_id=get_id(album))
+        return await self.get_paginated_list("albums", album_id=get_id(album))
```

Comment:
```suggestion
        return await self.post_relation("albums", album_id=get_id(album))
```
At least it seems to make more sense to me...

Reply:
Uhh yeah i dont know how i missed that

Reply:
I already noticed it and fixed it earlier actually, i dont know why github didnt register it:
Review: https://github.com/music-assistant/deezer-python-async/pull/1#discussion_r1288174852
Commit where i fixed it: [`b7e4568` (#1)](https://github.com/music-assistant/deezer-python-async/pull/1/commits/b7e4568755c74c1ad362fde367d2e3110524f8d3)